includes/admin/customers/customers.php

```diff
@@ -708,6 +711,9 @@ function edd_customers_view( $customer = null ) {
 								? esc_html( $address->address2 )
 								: '';
 							?>
+							<div class="row-actions">
```

Comment:
Adding the row actions does interesting things on mobile view because no cells are hidden:
![image](https://user-images.githubusercontent.com/4853066/86783363-33e69700-c02e-11ea-8d2d-5c2d4b27c29a.png)
(If the row/cell is not clicked/active, then there is just the gap.)